### ISSUE TITLE
Blurry icons in the Launcher at 400% Zoom

### ISSUE THREAD

**Issue by steff456** [jupyterlab/jupyterlab#12997]
## Description

I'm starting to work in the accessibility grant and did some audits for start identifying the issues in JupyterLab with 400% Zoom. One of them is that the icon for `Python 3` for both the Notebook and Console Launcher starts to blur with a lot of zoom,

**Default**

<img width="116" alt="image" src="https://user-images.githubusercontent.com/20992645/186281799-65a8a3f6-6d6b-4553-a35c-d187675a2220.png">

**200%**

<img width="221" alt="image" src="https://user-images.githubusercontent.com/20992645/186281775-4e29a21c-3af8-4504-980c-0f682b3b8721.png">

**400%**

<img width="1507" alt="image" src="https://user-images.githubusercontent.com/20992645/186281681-ee4e884a-5a71-408d-b97b-748c207e601d.png">

## Reproduce

1. Open the Launcher
2. Zoom in and check the Python 3 icon for the Notebook and Console 

## Expected behavior

I would expect to not have a blurry icon.

I want to make the contribution to fix this issue, but I was unable to find the asset that is loaded for this icon. In my investigation I discovered that apparently it is trying to load the icon from `/kernelspecs/python3/logo-64x64.png` and this one is different from the blue version that is at `packages/ui-components/style/icons/filetype/python.svg`. 

I hope you can guide me to make the PR and fix this issue! 

## Context

- Operating System and version: MacOS Catalina
- Browser and version: Chrome 104.0.5112.79 
- JupyterLab version: Latest master version

**Comment by tonyfast** [comment]
@steff456 very cool you are working on the zoom problem.

> I want to make the contribution to fix this issue, but I was unable to find the asset that is loaded for this icon.

the source images for kernels are actually in the kernel providers repository. the specific images for the python logo are in the `ipython/ipykernel` repo in this folder
https://github.com/ipython/ipykernel/blob/main/ipykernel/resources/

**Comment by krassowski** [comment]
This also affects other kernels notably IRkernel and Julia kernel:

![Screenshot from 2022-08-24 21-42-41](https://user-images.githubusercontent.com/5832902/186520724-3427d622-4598-4825-972c-df9d19340cc3.png)
